test/AdminManagerTests.java

```diff
@@ -50,246 +61,172 @@ public void testCreateAdminValid() {
         assertTrue("Original admin and loaded admin should share the same password",
                 loadedAdmin.comparePassword(password));
     }
+
+    /**
+     * Test invalid createAdmin with a username that already exists in the database.
+     */
     @Test(timeout = 1000)
     public void testCreateAdminInvalid() {
```

Comment:
Along with testing an invalid creation of an admin because the username already exists, maybe test an invalid creation of an admin because the username/password does not follow the new regex check in the regex-username-feature branch. Same goes for doctor, patient and secretary.

Reply:
Solved this in my latest commit :)